### ISSUE TITLE
Enhancement: Different Sprite Sheet Types for Character Select

### ISSUE THREAD

**Issue by IdealistCat** [FunkinCrew/Funkin#3731]
### Issue Checklist

- [X] I have properly named my enhancement
- [X] I have checked the Issues/Discussions pages to see if my enhancement has already been suggested

### What is your suggestion, and why should it be implemented?

Different spritesheet types for character select so that people won't need Adobe animate to make Playable character png and xml stuf, online tools wouldn't work

**Comment by Hundrec** [comment]
Are you proposing that the old type of spritesheets be compatible with character select for more accessible modding?

**Comment by Poii0** [comment]
> Are you proposing that the old type of spritesheets be compatible with character select for more accessible modding?

Yes that is what they are want for the select. But the only thing to happen is either A: The FNF spritesheet generator gets updated to break the sprite sheets like that or B: people have to pirate Adobe. But the idea isn't too bad though.

**Comment by IdealistCat** [comment]
> Are you proposing that the old type of spritesheets be compatible with character select for more accessible modding?

yes

**Comment by IdealistCat** [comment]
> > Are you proposing that the old type of spritesheets be compatible with character select for more accessible modding?
> 
> Yes that is what they are want for the select. But the only thing to happen is either A: The FNF spritesheet generator gets updated to break the sprite sheets like that or B: people have to pirate Adobe. But the idea isn't too bad though.

For A it would probably be better to have  a quickie little anim generator cause if it went for pixel by pixel finding what to break up and what not too it would be very difficult and slow.

**Comment by Starexify** [comment]
#6576 might close this ?
